torchmetrics/audio/pesq.py

```diff
@@ -0,0 +1,120 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Any, Callable, Optional
+
+from torch import Tensor, tensor
+
+from torchmetrics.functional.audio.pesq import pesq
+from torchmetrics.metric import Metric
+
```

Comment:
```suggestion
from torchmetrics.utilities.imports import _PESQ_AVAILABLE
```
we need a variable similar to these (to check that the package is available)
https://github.com/PyTorchLightning/metrics/blob/f9d7d5ff76b565956e25f639a755bed27241307c/torchmetrics/utilities/imports.py#L75-L84

Reply:
done